requirements.txt

```diff
@@ -1,3 +1,3 @@
 psutil>=3.2.2
-signalfx>=1.0.1
+signalfx>=1.0.4
```

Comment:
We typically don't break backwards compatibility. But we have in the past when we went from 0.x to 1.x for the python client lib. Given this, should we just do signafx==1.0.4.

@mdubbyap thoughts